server/src/http.rs

```diff
@@ -253,6 +258,30 @@ impl<Q: QueryExecutor + 'static> Service<Q> {
             })
     }
 
+    // GET /route/{table}
+    fn route(&self) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+        warp::path!("route" / String)
```

Comment:
warp::path!("debug" / "route")

Reply:
@chunshao90 I guess it should be uniform with `sql_query`, and that is to say, no debug prefix.